src/test/javascript/portal/details/LayerDetailsPanelSpec.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2012 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+describe("Portal.details.LayerDetailsPanel", function() {
```

Comment:
I think it's OK to have the file + the `describe` thing, but no need then for the `beforeEach` block.